pylint/checkers/spelling.py

```diff
@@ -383,8 +371,6 @@ def visit_functiondef(self, node):
             return
         self._check_docstring(node)
 
-    visit_asyncfunctiondef = visit_functiondef
```

Comment:
Is this intentional? This means we'll no longer analyze `AsyncFunctionDef` nodes.